infrastructure/Vagrantfile

```diff
@@ -7,7 +7,7 @@ Vagrant.configure("2") do |config|
     airy_core.vm.box_check_update = false
     airy_core.vm.hostname = "airy"
     airy_core.vm.network "private_network", ip: "192.168.50.5"
-    config.hostsupdater.aliases = %w[demo.airy api.airy]
+    config.hostsupdater.aliases = %w[assets.airy, demo.airy api.airy]
```

Comment:
in Ruby `%w[foo bar]` evaluates to `["foo", "bar"]` so the hostname here would have a comma